### PR TITLE
adds standard boilerplate

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
 - [smart contracts](smart-contracts/README.md)
+
+## How to write a standard?
+
+Create a `.md` file that follow this structure:
+
+```markdown
+# Wallet file format standard
+
+Initial meta issue: <https://github.com/massalabs/massa-standards/issues/{issue number}>
```
Comment (gregLibert) [inline README.md:16]:
```suggestion
Initial issue: <https://github.com/massalabs/massa-standards/issues/{issue number}>
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+## Specification
+
+## Implementation
+
+```
+
+Fill the issue number. The issue would probably be a meta issue. Sub-issues would be issue to implement
+the standard in proper repositories.
+
+Please use this VSCode extension for format markdown files
```
Comment (gregLibert) [inline README.md:31]: Why ?

File: wallet/dapps-communication.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Wallet-DApp communication standard
+
+Initial meta issue: <https://github.com/massalabs/massa-standards/issues/13>
```
Comment (gregLibert) [inline wallet/dapps-communication.md:3]:
```suggestion
Initial issue: #13
```

File: wallet/file-format.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Wallet file format standard
+
+Initial meta issue: <https://github.com/massalabs/massa-standards/issues/15>
```
Comment (gregLibert) [inline wallet/file-format.md:3]:
```suggestion
Initial issue: #15 
```

File: README.md [hunk 5]
```diff
@@ -1,5 +1,32 @@
...
 Massa standards
 
 - [smart contracts](smart-contracts/README.md)
+
+## How to write a standard?
+
+Create a `.md` file that follow this structure:
+
+```markdown
+# Wallet file format standard
```
[the remark this answers is not in the document]
Comment (Thykof) [reply README.md:14]: nope my bad

File: wallet/dapps-communication.md (hunk 3 above)
Comment (Thykof) [reply wallet/dapps-communication.md:3]: does not work in md file

File: wallet/file-format.md (hunk 4 above)
Comment (Thykof) [reply wallet/file-format.md:3]: does not work in md file

**Review by BatiGencho — APPROVED** [review]

**Pull request merged**